cadence/templates/server-job.yaml

```diff
@@ -282,7 +296,14 @@ spec:
             - name: SQL_USER
               value: {{ include "cadence.persistence.sql.user" (list $ $store) }}
             - name: SQL_PASSWORD
+              {{ - if .Values.mysql.enabled }}
```

Comment:
**Required**: here the typo needs to be fixed because it breaks the Helm chart linter CI check, apologies for missing it in the first place.

```suggestion
              {{- if .Values.mysql.enabled }}
```